### ISSUE TITLE
Internet Explorer Icons Very Large

### ISSUE THREAD

**Issue by mikeganz** [uikit/uikit#1336]
I have a basic portfolio site built - I used the uk-icons-large class to put some linked in and contact icons in... they are sized properly in safari, firefox and chrome - but a friend emailed me to tell me they were excessively large in IE11... 

the url is mikeganz.com/contact.html

I did a search, but came up empty - See the attached picture. Is this a known issue? 

![image001](https://cloud.githubusercontent.com/assets/12899109/8165837/585f2430-1359-11e5-8e75-333cbebe385e.png)

**Comment by malles** [comment]
Seems that IE takes the font-size of the `h1` the icon is in, 36px, and Chrome sticks to the 250% `uk-icon-large` size. Take the icon out of the `h1`.
You are loading `uikit.css` and `uikit.min.css`. That is is not helping it I'm afraid.

**Comment by mikeganz** [comment]
1) thank you, I'll work on it this evening. 2) crap - I forgot to clean up the "bloat" from CSS and JS that I have this thing rendering... thank you for the reminder. :+1:

**Comment by mikeganz** [comment]
It appears as though the problem was corrected by taking out the ( h1 ), as you suggested, and putting a ( p class="uk-h1" ) in its place. This allows the icon to remain as large as it was for the other browsers, but sized properly for IE. I also took out redundant CSS files. Now if I could just figure out how to use a navicon properly when it reaches the mobile breakpoint... hehe. Thank you for your assistance!
